src/kzg_multiproof.rs

```diff
@@ -94,12 +96,29 @@ fn verifier(proof: MultiProof) -> bool {
 mod tests {
     use crate::kzg_multiproof::{prover, verifier};
     use crate::kzg_tools::{polynomial::Polynomial, trusted_setup::trusted_setup};
+    use halo2::arithmetic::Field;
+    use halo2::halo2curves::bn256::Fr;
+    use rand::thread_rng;
 
     #[test]
     fn kzg_multiproof_test() {
         let p_committed = Polynomial::create_polynomial(7);
         let trusted_setup = trusted_setup(p_committed.clone());
-        let prover = prover(p_committed, trusted_setup.clone(), 3);
+
+        //Evaluate committed polynomial at determined number of points(signatures).
+        let k_points = 3;
+        let rng = thread_rng();
+        let mut z_points = Polynomial::new(Vec::new());
+        let mut y_points = Polynomial::new(Vec::new());
+
+        for _ in 0..k_points {
+            let random_num = Fr::random(rng.clone());
+            let eval = Polynomial::eval(&p_committed, random_num);
+            y_points.coeff.push(eval.evaluation);
```

Comment:
y points should be calculated inside the prover since prover only knows the polynomial